core/tests/test_utils.py

```diff
@@ -251,6 +282,7 @@ def signup(self, email, username):
         """Complete the signup process for the user with the given username."""
         self.login(email)
 
+        self.enable_urlfetch_mock()
```

Comment:
Some tests(mostly those related to map-reduce) don't work with this custom urlfetch mock. 
Errors like these come up --
* http://paste.ubuntu.com/15611897/
* http://paste.ubuntu.com/15611911/

So this mock is enabled only on signup, and disabled immediately once the signup is completed.